duckduckgo/duckduckgo-autofill
# Improve field scoring debugging

**Reviewer:**
**Asana:** https://app.asana.com/0/0/1204263809236636/f

## Description
Adds logs to debug field scoring behind a `sessionStorage` flag.
![CleanShot 2023-03-27 at 12 24 49@2x](https://user-images.githubusercontent.com/1828326/227915619-37066f68-a692-473f-9459-dad1e1e92ea4.jpg)


## Steps to test
In depth debugging instructions at [this page](https://app.asana.com/0/0/1204251534435626/f), but to see the logs just run:

```js
sessionStorage.setItem('ddg-autofill-debug', true)
```

To see a more complete set of log types visit https://www.roboform.com/filling-test-all-fields.
